.golangci.yml

```diff
@@ -19,3 +16,12 @@ linters:
 linters-settings:
   goimports:
     local-prefixes: github.com/openservicemesh/osm
+
+issues:
+  new: true
+  exclude-rules:
+    - linters:
+        - golint
```

Comment:
what is this doing?

Reply:
golint throws errors for the dot imports used for ginkgo/gomega. This ignores those errors.

Reply:
A comment inline will help.

Reply:
Just added

Reply:
Thank you